cmd/kconf.go

```diff
@@ -37,8 +37,8 @@ var rootCmd = &cobra.Command{
 
 var addCmd = &cobra.Command{
 	Use:   "add",
-	Short: "Add in a new kubeconfig file could set context name",
-	Long:  `Add a new kubeconfig file to the existing merged config file and set context name`,
+	Short: "Add in a new kubeconfig file and option context name",
+	Long:  `Add a new kubeconfig file to the existing merged config file and option context name`,
```

Comment:
These should be "optional" (proper grammar).

Reply:
Do you  mean like is` kconf add filepath --contextname context_name`?

Reply:
No, I meant the word "option" in the above text should be "optional"